METIS/METIS_LMS.yaml

```
@@ -8,7 +8,11 @@ date_modified: 2022-05-12
 changes:
   - 2022-03-04 (OC) introduce decouple_detector_from_sky_headers
   - 2022-03-22 (OC) add METISLMSImageSlicer effect
+<<<<<<< HEAD
   - 2022-04-13 (KL) added ATMO overrides for skycalc_use_local_file for ScopeSim v0.5 upgrade
+=======
+  - 2022-05-12 (OC) add fp2_platescale
+>>>>>>> dev_master
```

It seems you actually committed the conflicts! I mean, that seems to be what the commit message says, but probably not what you intended
```suggestion
  - 2022-04-13 (KL) added ATMO overrides for skycalc_use_local_file for ScopeSim v0.5 upgrade
  - 2022-05-12 (OC) add fp2_platescale
```